Modules/_xxsubinterpretersmodule.c

```diff
@@ -627,6 +627,36 @@ _channelends_associate(_channelends *ends, int64_t interp, int send)
     return 0;
 }
 
+static int64_t *
+_channelends_list_interpreters(_channelends *ends, int64_t *count, int send)
+{
+    int64_t *ids = NULL;
+    int64_t numopen;
+
+    assert(ends != NULL);
+    assert(count != NULL);
+    numopen = send ? ends->numsendopen : ends->numrecvopen;
+
+    if (numopen >= INT64_MAX) {
```

Comment:
`numopen` is an `int64_t` so there is no way this condition will ever be true, right?  (on top of that, there will never be more than `INT64_MAX` interpreters).